README.md

```diff
@@ -30,7 +30,26 @@ tiffutil -cathidpicheck cabal-desktop-dmg-background.jpg cabal-desktop-dmg-backg
 
 ## Distribute
 
-build for current platform:
+TravisCI will automatically create and upload the appropriate release packages
```

Comment:
@nikolaiwarner noting this

Reply:
thanks! this is great. atm i do have to manually build the mac version because i dont have travis set up with my keys for doing the notarizing.